OrbitGl/App.h

```diff
@@ -50,8 +50,8 @@ class OrbitApp : public CoreApp {
   void Inject(const std::string& file_name);
   virtual void StartCapture();
   virtual void StopCapture();
-  virtual void StartRemoteCaptureBufferingThread();
-  virtual void StopRemoteCaptureBufferingThread();
+  virtual void StartRemoteCaptureBufferingThread() override;
```

Comment:
As discussed, it might make sense to remove the redundant virtual keyword here.

Reply:
done